lib/rules/eol-last.js

```diff
@@ -22,6 +22,8 @@ module.exports = function(context) {
                 location = {column: 1},
                 linebreakStyle = context.options[0] || "unix",
                 linebreak = linebreakStyle === "unix" ? "\n" : "\r\n";
+
+            /* istanbul ignore if */
```

Comment:
I would remove this if statement as it is unreachable. Rules are not executed for empty files.